### DISCUSSION TITLE
CitraVR (Beta) v0.3

### DISCUSSION THREAD

**Discussion by amwatson** [amwatson/CitraVR#22] [Announcements]
## What's Changed
- Fixed crash for v62 users (Thanks @gyroninja for the fix)
- Added a "Settings > VR > CPU Level" setting to adjust CPU level (to prevent heat/battery drain)
- Fixed compilation errors when building debug 
- Dpad fix: adjusted default dpad bindings, which should hopefully fix the dpad for most gamepad users (note: you may have to bind the dpad in Settings this release).
- set CPU Clock Frequenct % to 100 and allowed for user adjustment
- Made app boundaryless (thanks @gyroninja)

## New Contributors
* @thestonefox made their first contribution in https://github.com/amwatson/CitraVR/pull/7
* @gyroninja made their first contribution in https://github.com/amwatson/CitraVR/pull/18

**Full Changelog**: https://github.com/amwatson/CitraVR/compare/v0.2.0...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/amwatson/CitraVR/releases/tag/v0.3.0'>CitraVR (Beta) v0.3</a>.</em>

**Comment by celeroo** [comment]
installed for the first time today, i get nothing but two black screens no matter which game i try to load

**Reply by celeroo** [reply]
ok i guess i was using encrypted roms, problem solved this thing is AMAAHHZING
